resources/resource_ids.spec

```diff
@@ -242,6 +242,10 @@
     "META": {"sizes": {"includes": [20]}},
     "includes": [34790],
   },
+  "<(SHARED_INTERMEDIATE_DIR)/brave/web-ui-email_aliases/email_aliases.grd": {
+    "META": {"sizes": {"includes": [100]}},
```

Comment:
100 seems larger than necessary? The actual number seems closer to 30?

Reply:
Currently 36, so I changed it to 40.